frontend/src/metabase/entities/actions/utils.ts

```diff
@@ -4,12 +4,11 @@ import type {
   FieldType,
   InputType,
   ParameterType,
-  ModelAction,
+  Parameter as ParameterObject,
```

Comment:
I'd recommend removing this change and rebasing your branches on the latest `master` since we've removed the entire `writeback` folder there